Cubical/Papers/ZCohomology.agda

```diff
@@ -156,7 +156,7 @@ open coHom using (K)
 open S using (sphereConnected)
 
 -- Lemma 1
-open S using (wedgeConSn)
+open S using (wedgeconFun; wedgeconLeft ; wedgeconRight)
 
 -- restated to match the formulation in the paper
 wedgeConSn' : ∀ {ℓ} (n m : ℕ) {A : (S₊ (suc n)) → (S₊ (suc m)) → Type ℓ}
```

Comment:
Same here, is it really necessary to quantify over `ℓ`?

Reply:
There are no variables in this file. I could of course add some if you want.